Add sentence about Set Configuration Fields (#5741)
segmentio/segment-docs: src/connections/destinations/catalog/actions-google-analytics-4-web/index.md

> 1. The **setConfigurationFields** mapping is enabled in your mappings.

Reviewer: ```suggestion
1. The **Set Configuration Fields** mapping is enabled in your mappings.
```

2. You've added at least one other event mapping for an event you want to send to Google Analytics.

> The **Set Configuration Fields** mapping is required for data to be sent downstream. This is because the **Set Configuration Fields** mapping sets configuration to Measurement ID and establishes data flow using the `config` command. If no other mappings are enabled, the GA4 destination will not send events downstream. 

Reviewer: ```suggestion
The **Set Configuration Fields** mapping is required for data to be sent downstream because it sets configuration to Measurement ID and establishes data flow using the `config` command. If you have no enabled mappings other than **Set Configuration Fields**, the GA4 destination does not send events downstream. 
```
